onnx/defs/gen_doc.py

```diff
@@ -163,9 +163,12 @@ def format_value(value: Any) -> str:
 
                 if isinstance(default_value, list):
                     default_value = [format_value(val) for val in default_value]
+                    opt = f"default is {default_value}"
+                elif hasattr(default_value, "doc_string"):
```

Comment:
Shouldn't the doc_string formatting be orthogonal to the default-value formatting? Shouldn't we generate both and concatenate them?

Reply:
I agree. Have done something to this effect.